test/e2e/docker-compose-base.yml

```diff
@@ -35,24 +35,11 @@ services:
       - GITHUB_BASE_REF=${GITHUB_BASE_REF}
       - GITHUB_HEAD_REF=${GITHUB_HEAD_REF}
       - SECRET_WORDS="orbit bench unit task food shock brand bracket domain regular warfare company announce wheel grape trust sphere boy doctor half guard ritual three ecology"
-    depends_on:
-      - display
-      - video
-      - agd
     entrypoint: []
```

Comment:
I initially planned to include `depends_on` in `docker-compose-base.yml`, but discovered that it's not a shared configuration across services:
https://docs.docker.com/compose/multiple-compose-files/extends/#exceptions-and-limitations